README.md

```diff
@@ -4,6 +4,25 @@ git clone https://github.com/theforeman/smart-proxy
 ```
 Configure smart proxy
 
+
+In smart-proxy directory 
+```shell
+mkdir smart-proxy/logs
+```
+Create two new files in smart-proxy/config/settings.d --> First one named `dynflow.yml` and second one `remote_execution_ssh.yml` and add lines  to both of them 
+```
+---
+:enabled: true
+``` 
+Then add a line that contains `:log_file: logs/proxy.log` to file `smart-proxy/config/settings.yml`
+
+In foreman directory add a line that contains `:restrict_registered_smart_proxies: false` to file `foreman/config/settings.yaml`
+
+Then create a file `Gemfile.local.rb`in directory `foreman/bundler.d/Gemfile.local.rb` and add line 
```

Comment:
It might be good to give the file a more descriptive name, something like `remote-execution.local.rb`

Anything matching the `*.local.rb` glob will be evaluated when running `bundle` and ignored by git